src/components/productDetails/list.jsx

```diff
@@ -45,6 +54,24 @@ const List = ({
     }
   }, [outfits, outfitsImages]);
 
+  function onArrowClick(direction) {
+    if (direction === "left") {
+      if (shownIndices[0] !== 0) {
+        let newShownIndices = shownIndices.map(idx => idx - 1);
+        setShownIndices(newShownIndices);
+      } else {
+        console.log("Already at left-most item");
```

Comment:
will this become an alert of some sort in production build?

Reply:
No it will be removed eventually. More for development purposes and making sure the arrows are functioning as expected. Nice catch. Will remove when no longer neeeded.